src/solidity-frontend/solidity_convert.cpp

```diff
@@ -1596,6 +1605,18 @@ bool solidity_convertert::get_elementary_type_name(
     new_type.set("#cpp_type", c_type);
     break;
   }
+  case SolidityGrammar::ElementaryTypeNameT::STRING:
+  {
+    size_t value_length = 128;
```

Comment:
Add some comments to explain why 128 is used?

Reply:
This here is a TODO as in actuality it should be the size of the string literals but we have not passed that information here

Reply:
Well, it's impossible to tell whether "128" is a TODO or not ;). Please be explicit.